includes/class-wc-privacy.php

```diff
@@ -369,8 +369,54 @@ protected static function delete_inactive_accounts_query( $timestamp, $limit = 2
 				require_once ABSPATH . 'wp-admin/includes/user.php';
 			}
 
+			$builtin_post_types = get_post_types( array( '_builtin' => true ) );
+			unset( $builtin_post_types['user_request'] ); // Exclude user_request from data migration.
+
+			/**
+			 * Filter post types that should be assign to an admin user prior user exclusion.
+			 *
+			 * @param array $post_types List of post types to assign to an admin. Defaults to WordPress builtin post types.
```

Comment:
```suggestion
			 * @since 5.3.0
			 * @param array $post_types List of post types to assign to an admin. Defaults to WordPress builtin post types.
```